mne_bids/read.py

```diff
@@ -109,7 +109,7 @@ def _handle_info_reading(sidecar_fname, raw, verbose=None):
 
     # read in the sidecar JSON's line frequency
     line_freq = sidecar_json.get("PowerLineFrequency")
-    if line_freq == "n/a":
+    if line_freq in ("n/a", -1):
```

Comment:
still I am surprise that the standard says that line_freq should be set to -1 and not 'n/a' when not available. Maybe it's more a validator issue?

@sappelhoff thoughts?

Reply:
that's definitely a validator issue. when missing, it should be `n/a`

very weird that the validator wants `-1`. @alexrockhill could you open an issue for that on the validator?

https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/03-electroencephalography.html#sidecar-json-_eegjson